src/collections/Breadcrumb/BreadcrumbDivider.js

```diff
@@ -18,7 +18,10 @@ function BreadcrumbDivider(props) {
   const rest = getUnhandledProps(BreadcrumbDivider, props)
 
   if (icon) {
-    return <Icon className={classes} name={icon} {...rest} />
+    // TODO: After update <Icon> to API replace with this code:
+    // return <Icon className={classes} name={icon} {...rest} />
+
+    return <Icon className={[icon, classes].join(' ')} {...rest} />
```

Comment:
Icon/Image props need to support both strings and components being passed.  We use the `iconPropRenderer` util for this.  This case is a little more tricky because we also need to add the users classes as well as the `divider` class to the icon after it is rendered.

The common test was over asserting things, I've updated it.  I've also pushed an update the prop renderers to support passing props.  I've tested this with the new renderers and it works:

```diff
-import { customPropTypes, getUnhandledProps } from '../../utils/propUtils'
-import Icon from 'src/elements/Icon/Icon'
+import { customPropTypes, getUnhandledProps, iconPropRenderer } from '../../utils/propUtils'

/**
 * A divider sub-component for Breadcrumb component.
 */
function BreadcrumbDivider(props) {
  const {
    children, icon, className,
  } = props
  const classes = cx(
    className,
    'divider',
  )
  const rest = getUnhandledProps(BreadcrumbDivider, props)

  if (icon) {
    // TODO: After update <Icon> to API replace with this code:
    // return <Icon className={classes} name={icon} {...rest} />

-  return <Icon className={[icon, classes].join(' ')} {...rest} />
+    const iconClasses = _.isString(icon)
+      ? cx(icon, classes)
+      : cx(icon.props.className, classes)
+
+    return iconPropRenderer(icon, { ...rest, className: iconClasses })
}
```

Note I also used `cx` to build up the icon classes.